CHANGELOG.md

```diff
@@ -5,13 +5,13 @@
 ```
 - <description> (#<PR_number>, kudos to @<author>)
 ```
+
+- added deep link generic for `Base.FlowCoordinator` (#47, kudos to @fortmarek)
```

Comment:
Please add it to **master** section 🙄 as your changes are gonna be merged (eventually) to **master** branch